amgut/db/patches/0041.sql

```diff
@@ -208,60 +282,239 @@ CREATE INDEX idx_dna_plate_extraction_robot_id ON pm.dna_plate ( extraction_robo
-CREATE INDEX idx_run_library_plate_library_plate_id ON pm.run_library_plate ( library_plate_id );
+    targeted_plate_id    bigint NOT NULL,
+    volume               real NOT NULL,
+    discarded            bool DEFAULT false NOT NULL,
+    CONSTRAINT pk_targeted_pool PRIMARY KEY ( targeted_pool_id ),
+    CONSTRAINT idx_targeted_pool UNIQUE ( name ),
+    CONSTRAINT fk_target_gene_pool FOREIGN KEY ( targeted_plate_id ) REFERENCES pm.targeted_plate( targeted_plate_id )
+ ) ;
+ CREATE INDEX idx_target_gene_pool_0 ON pm.targeted_pool ( targeted_plate_id ) ;
+
+CREATE TABLE pm.shotgun_plate (
+    shotgun_plate_id            bigserial  NOT NULL,
+    name                        varchar  NOT NULL,
+    email                       varchar  NOT NULL,
+    created_on                  timestamp  NOT NULL,
+    processing_robot_id         bigint  NOT NULL,
+    plate_type_id               bigint  NOT NULL,
+    volume                      real  NOT NULL,
+    dna_quantification_date     timestamp  ,
+    dna_quantification_email    varchar  ,
+    dna_quantification_volume   real  ,
+    plate_reader_id             bigint  ,
+    CONSTRAINT pk_shotgun_plate PRIMARY KEY ( shotgun_plate_id ),
+    CONSTRAINT idx_shotgun_plate UNIQUE ( name ),
+    CONSTRAINT fk_shotgun_plate FOREIGN KEY ( email ) REFERENCES ag.labadmin_users( email ),
+    CONSTRAINT fk_shotgun_plate_prc_robot FOREIGN KEY ( processing_robot_id ) REFERENCES pm.processing_robot( processing_robot_id ),
+    CONSTRAINT fk_shotgun_plate_type FOREIGN KEY ( plate_type_id ) REFERENCES pm.plate_type( plate_type_id ),
+    CONSTRAINT fk_shotgun_plate_email FOREIGN KEY ( dna_quantification_email ) REFERENCES ag.labadmin_users( email ),
+    CONSTRAINT fk_shotgun_plate_reader FOREIGN KEY ( plate_reader_id ) REFERENCES pm.plate_reader( plate_reader_id )
+ ) ;
+CREATE INDEX idx_shotgun_plate_0 ON pm.shotgun_plate ( email ) ;
+CREATE INDEX idx_shotgun_plate_1 ON pm.shotgun_plate ( processing_robot_id ) ;
+CREATE INDEX idx_shotgun_plate_2 ON pm.shotgun_plate ( plate_type_id ) ;
+CREATE INDEX idx_shotgun_plate_3 ON pm.shotgun_plate ( dna_quantification_email ) ;
+CREATE INDEX idx_shotgun_plate_4 ON pm.shotgun_plate ( plate_reader_id ) ;
+
+CREATE TABLE pm.condensed_plates (
+    shotgun_plate_id  bigint  NOT NULL,
+    dna_plate_id           bigint  NOT NULL,
+    position               integer  NOT NULL,
+    CONSTRAINT idx_condensed_plates PRIMARY KEY ( shotgun_plate_id, dna_plate_id, position ),
+    CONSTRAINT fk_condensed_plates FOREIGN KEY ( shotgun_plate_id ) REFERENCES pm.shotgun_plate( shotgun_plate_id ),
+    CONSTRAINT fk_condensed_plates_dna_plate FOREIGN KEY ( dna_plate_id ) REFERENCES pm.dna_plate( dna_plate_id )
+ ) ;
+CREATE INDEX idx_condensed_plates_0 ON pm.condensed_plates ( shotgun_plate_id ) ;
+CREATE INDEX idx_condensed_plates_1 ON pm.condensed_plates ( dna_plate_id ) ;
+
+CREATE TABLE pm.shotgun_plate_layout (
```

Comment:
No, the reason why I need another layout here is because the shotgun plate in general is a 384-well plate but in the targeted case we are using the same layout as in the sample plate